app/views/mechanics/show.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1>Mechanic Info</h1>
+<p>Name: <%= @mechanic.name %></p>
+<p>Years of Experience: <%= @mechanic.years_experience %></p>
+<section id="mechanic_rides">
+  <h3>Rides Worked On</h3>
+  <% @mechanic.rides.each do |ride| %>
+    <section id= "ride-<%= ride.id %>">
+      <p><%= ride.name %></p>
+    </section>
+  <% end %>
+</section>
+
+<%= form_with url: "/mechanics/#{@mechanic.id}/rides", method: :post, data: { turbo: false } do |form| %>
+  <%= form.label :"Add a Ride by ID Number" %>
+  <%= form.text_field :ride_id %>
+  <%= form.submit :Submit %>
+<% end %>
```

Comment:
Wow, this is great. The form is submitting to the correct path, using the correct verb. 🙌